src/relay.rs

```diff
@@ -293,6 +376,144 @@ impl<T: DuplexTransport + 'static> Network<T> {
+        let tx_hash = transfer.tx_hash.clone();
+        let block_hash = transfer.block_hash.clone();
+        let block_number: &mut[u8] = &mut vec![0; 32];
+        transfer.block_number.to_big_endian(block_number);
+        let mut grouped: Vec<u8> = Vec::new();
+        grouped.extend_from_slice(&tx_hash[..]);
+        grouped.extend_from_slice(&block_hash[..]);
+        grouped.extend_from_slice(block_number);
+        let hash = H256(keccak256(&grouped[..]));
+        let account = self.account.clone();
+        self.relay.query::<Withdrawal, Address, BlockNumber, H256>(
+                "withdrawals",
+                hash,
+                account,
+                Options::default(),
+                BlockNumber::Latest,
+        ).or_else(|e| {
+            error!("error getting withdrawal: {}", e);
+            Err(())
+        })
+    }
+
+    /// Returns a Stream of Transfer that were missed, either from downtime, or failed approvals
+    ///
+    /// # Arguments
+    ///
+    /// * `handle` - Handle to the event loop to spawn additional futures
+    pub fn missed_transfer_stream(&self, interval: reactor::Interval, handle: &reactor::Handle) -> impl Stream<Item = Transfer, Error = ()> {
+        let (tx, rx) = mpsc::unbounded();
+        let handle = handle.clone();
+        let h = handle.clone();
+        let token_address = self.token.address().clone();
+        let relay_address = self.relay.address().clone();
+        let network_type = self.network_type;
+        let web3 = self.web3.clone();
+        let future = interval.for_each(move |_| {
+            let handle = h.clone();
+            let tx = tx.clone();
+            let web3 = web3.clone();
+            let w3 = web3.clone();
+            web3.clone().eth().block_number()
+                .and_then(move |block| {
+                    Ok(FilterBuilder::default()
+                        .address(vec![token_address])
+                        .from_block(BlockNumber::from(block.as_u64() - 100))
+                        .to_block(BlockNumber::from(block.as_u64() - 25))
+                        .topics(
+                            Some(vec![TRANSFER_EVENT_SIGNATURE.into()]),
+                            None,
+                            Some(vec![relay_address.into()]),
+                            None,
+                        ).build())
+                }).and_then(move |filter| {
+                    let web3 = web3.clone();
+                    web3.eth().logs(filter.clone())
+                }).and_then(move |logs| {
+                    let tx = tx.clone();
+                    let web3 = w3.clone();
+                    let handle = handle.clone();
+                    info!("found {} log(s)", &logs.len());
+                    logs.iter().for_each(|log| {
+                        if Some(true) == log.removed {
+                            info!("removed logs :(");
```

Comment:
Probably should be a warn! (might reconsider message too), and also we don't want to break early if one log is removed right?